CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8.8)
+cmake_minimum_required (VERSION 3.0)
```

Comment:
I will think about if we do the cmake 3 version push this release or next one. For Debian it should be actually okay because it only effects oldoldstable.